### ISSUE TITLE
Context menu starts multiple instances on Windows

### ISSUE THREAD

**Issue by skombijohn** [dosbox-staging/dosbox-staging#3429]
### Are you using the latest Dosbox-Staging Version?

- [X] I have checked releases and am using the latest release.

### Different version than latest?

0.81.0

### What Operating System are you using?

Windows 10

### Whats your question and how can we help?

Hi 

i am a experienced DosBOX user, and recently discovered DosBOX Staging. 
I LOVE the CRT emulation, brings back so many memories of the past :)

I also like the approach of having "game folders" with completely setup drives etc. without having to manage multiple games and settings in one big dosdrive - which was my approach in the past.

I followed the tutorial for PrinceOfPersia - everything works of course.

But:
When I start DosBox Staging from the windows context menu in or "on" the folder, it will always startup multiple dosbox windows, all starting the game (or whatever is in the autoexec section).
I installed second game in parallel, now sometimes multiple games get started in parallel.

I am not sure what could have went wrong.

I would appreciate any help.

Thanks

### Code of Conduct & Contributing Guidelines

- [X] Yes, I agree.

**Comment by johnnovak** [comment]
This is very strange and I can't replicate it @skombijohn. I've been uninstalling and reinstalling Staging many times today when testing the Windows installer on Win 10 and everything works as expected.

I'm inclined to close this unless you can provide some exact reproduction steps.

**Comment by johnnovak** [comment]
Can't reproduce. Something weird must be going on on the guy's setup, but I don't think we have an issue that needs fixing.

**Comment by skombijohn** [comment]
@johnnovak 
Sorry for late response, just wanted to provide some feedback:

This was probably due some weird strange issue in my 5+ year old Windows 10 installation. I noticed whenever I open a cmd window from e.g. explorer address bar, TWO cmd windows popped up. 
It was very likely not related to DOSBos Staging.

I did a complete OS reinstall (was about time).
DOSBox Staging behaves very well now :)

Thanks!
